templates/CRM/Event/Form/ManageEvent/Tab.tpl

```diff
@@ -107,3 +107,4 @@ CRM.$(function($) {
 </script>
 {/literal}
 {include file="CRM/Event/Form/ManageEvent/ConfirmRepeatMode.tpl" entityID=$id entityTable="civicrm_event"}
+{include file="CRM/common/TabSelected.tpl" defaultTab="settings"}
```

Comment:
@mattwire @eileenmcnaughton Currently this line breaks the manage event URLs if selectedChild is not appended to it. Eg

https://dmaster.demo.civicrm.org/civicrm/event/manage/fee?reset=1&action=update&id=3 goes to the settings page instead of opening the `fee` tab. Similarly all the configuration links from the manage event page goes to the settings page.

Maybe, we should just 

- remove this line from the file or 
- append selectedChild to the links in Manage Event Page. But we already specify the tab name in the link before the query string eg `civicrm/event/manage/fee?reset=1`. 
- Format the links to look like `civicrm/event/manage?..selectedChild=fee` (might need xml/upgrade changes along with fixing the links in core).

Reply:
@mattwire @jitendrapurohit maybe we should assign defaultTab from php?

@mattwire I think this needs to be fixed in the rc

Reply:
Ok. So before this PR the selectedChild stuff didn't work for events at all.  Events is the only one that specifies the tab as part of the link.

> 1. remove this line from the file

Specifying the "defaultTab" is standard across all usages of the tab selector code and is meant to be a fallback default if selectedChild is not defined.

2. append selectedChild to the links in Manage Event Page. But we already specify the tab name in the link before the query string eg civicrm/event/manage/fee?reset=1.

This is probably the easiest / safest way to resolve this, unless we assign defaultTab from PHP in 1. 

> 3. Format the links to look like civicrm/event/manage?..selectedChild=fee (might need xml/upgrade changes along with fixing the links in core).

This would be more standard and is what we should probably aspire to for consistency, but is also much more likely to cause breakage.